swan/values.yaml

```diff
@@ -166,6 +166,7 @@ jupyterhub:
         # set home directory to EOS
         local_home: False
         centos7_image: "gitlab-registry.cern.ch/swan/docker-images/systemuser:v6.0.0"
+        accpy_image: "gitlab-registry.cern.ch/swan/docker-images/jupyter/swan-acc:v0.0.1"
```

Comment:
Once you have done the changes I requested for its Dockerfile, you could bump this version to `v1.0.0`